samples/event-logger/src/components/InputBoard/InputBoard.js

```diff
@@ -64,15 +82,21 @@ const InputBoard = ({className}) => {
 	const syntheticEventOnRef = useRef();
 	const timerIndexRef = useRef();
 
-	const activeEvents = useSelector((state) => state.activeEvents);
-	const eventCapturingOn = useSelector((state) => state.eventCapturingOn);
-	const eventLogs = useSelector((state) => state.eventLogs);
-	const syntheticEventOn = useSelector((state) => state.syntheticEventOn);
-	const timerIndex = useSelector((state) => state.timerIndex);
-
-	const onAddEventLog = useCallback((timeoutId, eventName, isDOMElement, isCapturing, eventObject) => dispatch(addEventLog(timeoutId, eventName, isDOMElement, isCapturing, eventObject)), [dispatch]);
-	const onRemoveEventLog = useCallback((eventName, isDOMElement, isCapturing) => dispatch(removeEventLog(eventName, isDOMElement, isCapturing)), [dispatch]);
-	const onUpdateEventLog = useCallback((prevTimeoutId, postTimeoutId, eventObject) => dispatch(updateEventLog(prevTimeoutId, postTimeoutId, eventObject)), [dispatch]);
+	const onAddEventLog =
+		useCallback(
+			(timeoutId, eventName, isDOMElement, isCapturing, eventObject) =>
```

Comment:
Please check the order of the parameters

Reply:
Modified to match the parameter order of the action function.